modules/ccalib/src/omnidir.cpp

```diff
@@ -1995,52 +1993,6 @@ double cv::omnidir::internal::computeMeanReproErrStereo(InputArrayOfArrays objec
     return reProErr;
 }
 
-void cv::omnidir::internal::checkFixed(Mat& G, int flags, int n)
```

Comment:
It should be dropped from `.hpp` file too.

Reply:
@alalek fixed